app/database/database_setup.py

```diff
@@ -0,0 +1,124 @@
+import sqlite3
+
+from app.util.conf.configuration import Configuration
+
+
+class FailedSQLiteTableSetup(Exception):
+    pass
+
+
+class DatabaseSetup:
```

Comment:
This does not seem like something that should be a class. I'm not sure we need this at all since sqlalchemy can do creates, but if it is needed I'd rather the methods in this class just be functions inside the module (not wrapped in a class).